lib/gooddata/bricks/user_filters_brick.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'brick'
+
+module GoodData::Bricks
```

Comment:
Similar code found in provisioning_brick.rb:3, release_brick.rb:3, rollout_brick.rb:3, user_filters_brick.rb:3, users_brick.rb:3 (mass = 85)